docs/tensorrt_plugin.md

```diff
@@ -0,0 +1,176 @@
+# TensorRT Plugins for custom operators in MMCV (Experimental)
+
+<!-- TOC -->
+
+- [TensorRT Plugins for custom operators in MMCV (Experimental)](#tensorrt-plugins-for-custom-operators-in-mmcv-experimental)
+  - [Introduction of TensorRT](#introduction-of-tensorrt)
+  - [Why include TensorRT plugins in MMCV](#why-include-tensorrt-plugins-in-mmcv)
+  - [List of TensorRT plugins supported in MMCV](#list-of-tensorrt-plugins-supported-in-mmcv)
+  - [How to build TensorRT plugins in MMCV](#how-to-build-tensorrt-plugins-in-mmcv)
+    - [Prerequisite](#prerequisite)
+    - [Build on Linux](#build-on-linux)
+  - [Create TensorRT engine and run inference in python](#create-tensorrt-engine-and-run-inference-in-python)
+  - [How to add a TensorRT plugin for custom op in MMCV](#how-to-add-a-tensorrt-plugin-for-custom-op-in-mmcv)
+    - [Main procedures](#main-procedures)
+    - [Reminders](#reminders)
+  - [Known Issues](#known-issues)
+  - [References](#references)
+
+<!-- TOC -->
+
+## Introduction of TensorRT
+
+**NVIDIA TensorRT** is a software development kit(SDK) for high-performance inference of deep learning models. It includes a deep learning inference optimizer and runtime that delivers low latency and high-throughput for deep learning inference applications. Please check its [developer's website](https://developer.nvidia.com/tensorrt) for more information.
+
+## Why include TensorRT plugins in MMCV
```

Comment:
May merge this section into the introduction section as there is only one sentence?